build.gradle

```diff
@@ -70,8 +70,8 @@ dependencies {
 sourceCompatibility = 1.11
 targetCompatibility = 1.11
 
-if (sourceCompatibility != Jvm.current().javaVersion) {
```

Comment:
We could also just remove this check and let java handle this. It would print something like
`Could not target platform: 'Java SE 11' using tool chain: 'JDK 8 (1.8)'.`

Reply:
We initially did not have this check, but a few devs reached out to us to ask why they couldn't build, which means that the default error message and a note in the README wasn't sufficient.